gcloudsubmit2.sh

```diff
@@ -9,4 +9,10 @@ echo $JAVA_HOME
9.Final.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/netty-all-4.0.23.Final.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/netty-all-4.1.17.Final.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/nimbus-jose-jwt-4.41.1.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/ojalgo-43.0.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/okhttp-2.7.5.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/okio-1.6.0.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/paranamer-2.3.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/paranamer-2.8.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/parquet-jackson-1.10.1.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/protobuf-java-2.5.0.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/servlet-api-2.5.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/servlet-api.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/slf4j-api-1.7.25.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/slf4j-log4j12-1.7.25.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/snappy-java-1.0.5.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/sparksftpTest-assembly-0.0.1.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/stax-api-1.0-2.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/stax2-api-3.1.4.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/tomcat-coyote.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/tomcat-dbcp.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/tomcat-i18n-es.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/tomcat-i18n-fr.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/tomcat-i18n-ja.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/tomcat-juli.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/woodstox-core-5.0.3.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/xercesImpl-2.9.1.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/xml-apis-1.3.04.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/xmlenc-0.52.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/xz-1.0.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/zookeeper-3.4.6.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/jsch-0.1.53.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/datanucleus-api-jdo-3.2.6.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/datanucleus-core-3.2.10.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/datanucleus-core-3.2.10-sources.jar,gs://dataproc-419763b0-894e-4402-8a6d-ed84cf766202-europe-west1/datanucleus-rdbms-3.2.9.jar
```

Comment:
Could we keep just one submit for the sake of simplicity?

Reply:
There is one for each implementation.
- SFTP with Hadoop SFTP Filesystem, dynamic allocation
- SFTP with Hadoop SFTP Filesystem, fixed allocation of executors
- SFTP with the SFTP connector, dynamic allocation
- SFTP with the SFTP connector, fixed allocation of executors.

To do the tests, would be more complex, is that your request anyway?

Reply:
This give me the idea that  we can simplify the scripts, putting the common parts in one, but keeping the different parameters in different scripts.

Reply:
Changed in 1c50ce0d072c38f15ea6b6e0a7b986585ad9daff